File: python/7/2577.py
a = int(input())
b = int(input())
c = int(input())

number = str(a * b * c)

frequency = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]

for letter in number:
    frequency[int(letter)] += 1

for count in frequency:
    print(count)

'''
number = a * b * c

frequency = [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
while number:
    wkfltn = number % 10
    frequency[wkfltn] += 1

    number //= 10

for count in frequency:
    print(count)
    '''

File: c/7/2577.c
#include <stdio.h>

int main() {
  int a, b, c;
  scanf("%d %d %d", &a, &b, &c);

  int number = a * b * c;
  int count[10] = {};
  while (number > 0) {
    count[number % 10]++;
    number /= 10;
  }

  for (int i = 0; i < 10; i++) {
    printf("%d\n", count[i]);
  }

  return 0;
}
